src/class/audio/audio_device.h

```diff
@@ -217,6 +217,8 @@ tu_fifo_t* tud_audio_n_get_ep_out_ff   (uint8_t func_id);
 uint16_t   tud_audio_n_write          (uint8_t func_id, const void * data, uint16_t len);
 bool       tud_audio_n_clear_ep_in_ff (uint8_t func_id);
 tu_fifo_t* tud_audio_n_get_ep_in_ff   (uint8_t func_id);
+uint16_t   tud_audio_n_get_ep_in_target_fifo_size(uint8_t func_id);
+void       tud_audio_n_set_ep_in_target_fifo_size(uint8_t func_id, uint16_t target_fifo_size);
```

Comment:
IMO `tud_audio_n_set_ep_in_fifo_threahold(uint8_t func_id, uint16_t fifo_threshold);`  is a better naming